docs/integration.md

```diff
@@ -0,0 +1,106 @@
+# Integrate with Rafiki
+
+**❗ Rafiki is intended to be run by [Account Servicing Entities](./glossary.md#account-servicing-entity) only and should not be used in production by non-regulated entities.**
+
+Account Servicing Entities provide and maintain payment accounts. In order to make these accounts Interledger-enabled via Rafiki, they need to provide the following endpoints and services.
+
+## Quotes / Fees
+
+Every Interledger payment is preceded with a quote that estimates the costs for transfering value from A to B. The Account Servicing Entity may charge fees on top of that for facilitating that transfer. How they strucutre those fees is completely up to the Account Servicing Entity.
+
+The Account Servicing Entity is required to provide an endpoint that is accessible to the Rafiki backend. It accepts a `POST` request with
+
+#### Request Body
+
+| Variable Name             | Type                                     | Description                                                                   |
+| ------------------------- | ---------------------------------------- | ----------------------------------------------------------------------------- |
+| id                        | string                                   | Interledger quote id                                                          |
+| paymentType               | enum: `'FixedSend'` \| `'FixedDelivery'` | fixed-send or fixed-receive payment                                           |
+| paymentPointerId          | string                                   | id of sending payment pointer                                                 |
+| receiver                  | string                                   | receiving payment pointer                                                     |
+| sendAmount                | [Amount](#amount)                        | defined or quoted send amount                                                 |
+| receiveAmount             | [Amount](#amount)                        | defined or quoted receive amount                                              |
+| maxPacketAmount           | number // bigint                         | discovered maximum packet amount allowed over chosen Interledger payment path |
+| minExchangeRate           | number                                   | aggregate exchange rate the payment is guaranteed to meet                     |
+| lowEstimatedExchangeRate  | number                                   | lower bound of probed exchange rate over the chosen Interledger payment path  |
+| highEstimatedExchangeRate | number                                   | upper bound of probed exchange rate over the chosen Interledger payment path  |
```

Comment:
```suggestion
```
https://github.com/interledger/rafiki/blob/a6db21310be34ee1997bfb9f87a361d50ffe3cfb/packages/backend/src/open_payments/quote/service.ts#L295-L300
https://github.com/interledger/rafiki/blob/a6db21310be34ee1997bfb9f87a361d50ffe3cfb/packages/backend/src/open_payments/quote/model.ts#L141-L158
https://github.com/interledger/rafiki/blob/a6db21310be34ee1997bfb9f87a361d50ffe3cfb/packages/backend/src/open_payments/quote/service.test.ts#L178-L198

Reply:
That makes a lot of sense and I can't read code... 🤦‍♀️